Windows: Build & Test issues

# Bug Report

Apparently when running on Windows we run into type inference issues when declarations are set to true. 

```
ERROR in libs/barista-components/expandable-section/src/expandable-section.ts:83:32 - error TS2742: The inferred type of '_sectionExpanded' cannot be named without a reference to 'C:/workspaces/barista/node_modules/rxjs'. This is likely not portable. A type annotation is necessary.
83   @Output('expanded') readonly _sectionExpanded = this.expandChange.pipe(
                                  ~~~~~~~~~~~~~~~~
libs/barista-components/expandable-section/src/expandable-section.ts:88:33 - error TS2742: The inferred type of '_sectionCollapsed' cannot be named without a reference to 'C:/workspaces/barista/node_modules/rxjs'. This is likely not portable. A type annotation is necessary.
88   @Output('collapsed') readonly _sectionCollapsed = this.expandChange.pipe(
```

Also the snapshot serializer causes issues on windows when paths are serialized within the snapshot, e.g. for schematics that generate a file tree.

```
...
                  "styles": Array [
    -               "node_modules/@dynatrace/barista-fonts/typography.scss",
    -               "node_modules/@dynatrace/barista-components/style/index.scss",
    +               "node_modules\\@dynatrace\\barista-fonts\\typography.scss",
    +               "node_modules\\@dynatrace\\barista-components\\style\\index.scss",
                    "apps/myapp/src/styles.scss",
                  ],
...
```

--- a/libs/barista-components/schematics/jest.config.js
+++ b/libs/barista-components/schematics/jest.config.js
@@ -2,4 +2,5 @@ module.exports = {
   name: 'schematics',
   preset: '../../../jest.config.js',
   coverageDirectory: '../../../coverage/components/schematics',
+  snapshotSerializers: ['jest-serializer-path'],
 };

--- a/package.json
+++ b/package.json
@@ -130,6 +130,7 @@
     "jest": "^24.8.0",
     "jest-junit": "^6.4.0",
     "jest-preset-angular": "8.0.0",
+    "jest-serializer-path": "^0.1.15",
     "markdown-it": "^10.0.0",
     "markdown-it-deflist": "^2.0.3",
     "memfs": "^3.0.3",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -3,7 +3,7 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": ".",
-    "declaration": true,
+    "declaration": false,
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "importHelpers": true,

